cdap-client/src/main/java/co/cask/cdap/client/ServiceClient.java

```diff
@@ -199,12 +199,12 @@ public String getAvailability(ServiceId service) throws IOException, Unauthentic
    * @throws NotFoundException @throws NotFoundException if the app or service could not be found
    * @throws IOException if a network error occurred
    * @throws UnauthenticatedException if the request is not authorized successfully in the gateway server
-   * @deprecated since 4.0.0. Please use {@link #getServiceURL(ProgramId)}  instead
    */
-  @Deprecated
```

Comment:
Actually we don't need version for routeconfig, since versions are contained in the content of routeconfig, but this change should be made for start/stop program. Routeconfig commands also need to be rearranged.